framework/raw-aes-keyring.md

```diff
@@ -196,19 +197,19 @@ If decrypting, the keyring MUST use AES-GCM with the following specifics:
 - it uses the [authentication tag](#authentication-tag) obtained from deserialization as the AES-GCM input authentication tag.
 - it uses this keyring's [wrapping key](#wrapping-key) as the AES-GCM cipher key.
 - it uses the [IV](#iv) obtained from deserialization as the AES-GCM IV.
-- it uses the encryption context from the [decryption materials](#structures.md#decryption-materials) as the AES-GCM AAD.
+- it uses the encryption context from the input as the AES-GCM AAD.
 - the AAD is serialized in the same format as the serialization of [message header AAD key value pairs](#message-header.md#key-value-pairs)
 
 If a decryption succeeds, this keyring MUST:
 
 - append a new [record](#structures.md#record) to the [keyring trace](#structures.md#keyring-trace)
-  in the input [decryption materials](#structures.md#decryption-materials).
+  in the output [data key materials](#structures.md#data-key-materials).
   This record MUST contain this keyring's [key name](#key-name) and [key namespace](#key-namespace),
   and the [flags](#structures.md$flags) field of this record MUST include the following flags:
   - [DECRYPTED DATA KEY](#structures.md#supported-flags)
-- add the resulting plaintext data key to the decryption materials and return the modified materials.
+- specify the resulting plaintext data key in the output [data key materials](#structures.md#data-key-materials).
```

Comment:
s/specify/set

Reply:
I think I need different language - "set" implies mutating an existing object too strongly, so I worry about confusing the reader. I'm not sure what would be better though. Perhaps "include"?

Reply:
I like includes :+1: